### ISSUE TITLE
How long should an epoch last?

### ISSUE THREAD

**Issue by bobogumizelka** [NeuroSYS-pl/objects_counting_dmap#6]
Hi!
Ifind you're work interesting, I'd like to test it. I have downloaded mall dataset with your get_data.py and I started to train with train.py. It's been 2 or 3 hours already, and all I can see in console is "Epoch 1". How long (aprox) should an epoch take? Is it possible it takes so long? Or I do something wrong?

**Comment by tgolan-neurosys** [comment]
A single epoch should run (on GPU) for about 10 minutes. If you have to measure epoch's time in hours there is definitely something wrong (unless you run training on CPU?).

**Comment by bobogumizelka** [comment]
Now the net is training fast - I train on Google Colab, so it was probably some Colab problem, couple day later everything worked fine. 
I trained UNet on mall dataset using your dataloaders. I trained for 150 epochs. After fist epoch I got:
Train:
	Average loss: 0.0827
	Mean error: -1.467
	Mean absolute error: 35.911
	Error deviation: 100.750
<Figure size 432x288 with 0 Axes>
Valid:
	Average loss: 0.0847
	Mean error: -0.468
	Mean absolute error: 3.579
	Error deviation: 4.358
New best result: 3.5789928784179694
Then, after 38 epochs I got:
Train:
	Average loss: 0.0668
	Mean error: -0.264
	Mean absolute error: 6.224
	Error deviation: 7.742
<Figure size 432x288 with 0 Axes>
Valid:
	Average loss: 0.0784
	Mean error: 1.844
	Mean absolute error: 3.202
	Error deviation: 3.476
New best result: 3.2020597070312538
After that there was no more improvement.
I run predictions for sanity test on mall dataset, but the results make no sens:
https://user-images.githubusercontent.com/37378492/74016522-da047700-4992-11ea-88f6-425bdf74b07c.JPG
